app/javascript/components/PostEditor.js

```diff
@@ -37,23 +39,50 @@ class PostEditor extends React.Component {
 		// TODO: should prevent closing tab while hasChanges state is true
 
 		this.state = {
-			// don't copy props.post into state unless we expect to update it
+      post: this.props.post || null,
 			isLoading: false, // is sending request to server
 			hasChanges: false, // editor has changes (actual onChange may be debounced)
 			error: null, // last server error, if any
 			errorAt: null, // string or null
 			lastSavedAt: getTimestamp('updated_at', props.post), // string or null
 			lastUnsavedChangeAt: null, // Date object or null, used to track dirty state
+      isProcessing: this.props.isProcessing || false,
 		}
 	}
 
+  componentDidMount(){
+    var post = document.getElementById('post').getAttribute("data-post-id")
+    var cableHost = this.state.post.data.attributes.cable_url
+    var cable = createConsumer(cableHost)
+
+    cable.subscriptions.create({channel: "PostsChannel", post_id: post}, {
+      connected() {
+        console.log("connected to PostsChannel")
```

Comment:
there are 3 console.log in here. are these left here intentionally?

Reply:
Removed two of them.